rebasehelper/utils.py

```diff
@@ -323,7 +322,27 @@ def close(self):
         session = requests.Session()
         session.mount('ftp://', FTPAdapter())
 
-        r = session.get(url, stream=True)
+        try:
+            return session.get(url, **kwargs)
+        except requests.exceptions.RequestException as e:
+            logger.error('%s: %s', type(e).__name__, six.text_type(e))
+            sys.stdout.write('%s: %s' % (type(e).__name__, six.text_type(e)))
```

Comment:
Ok, this time it's the same message printed to output twice. You should probably get familiar with how logging works in this project. In short, unless `--verbose` is specified, anything with level [`logging.INFO` or above](https://docs.python.org/3/library/logging.html#logging-levels) will be both printed to console and written to debug log file.

Reply:
Oh, sorry, I didn't know that. So just logger.error() should be enough to write it to debug log file and inform the user. I will fix it